src/com/company/Main.java

```diff
@@ -8,18 +8,49 @@
 
 
 public class Main {
-    public static void main(String[] args){
+    public static void main(String[] args) {
+
+
+        String txtFilePath = null;
+        try {
+            txtFilePath = args[0];
+        } catch (Exception e) {
+            System.out.println("Path has not entered");
+            System.exit(1);
```

Comment:
Return statement instead of exit() ?

Reply:
I have replaced it with return statement.